### PR TITLE
sentry: Migrate from raven to sentry-sdk

### PR DESCRIPTION
**Pull request by torotil**

### CODE REVIEW

File: moflask/flask.py [hunk 1]
```diff
@@ -43,11 +43,13 @@ def init_defaults(self):
...
-            # pylint: disable=import-outside-toplevel,import-error
-            from raven.contrib.flask import Sentry
+        # pylint: disable=import-outside-toplevel
+        if config := self.config.get("SENTRY_CONFIG"):
+            import sentry_sdk
+            from sentry_sdk.integrations.flask import FlaskIntegration
 
-            self.sentry = Sentry(self)
+            defaults = {"integrations": [FlaskIntegration()]}
+            self.sentry = sentry_sdk.Client(**{**defaults, **config})
```
Comment (katzmo) [inline moflask/flask.py:52]: I think `sentry_sdk.Client(**defaults, **config)` would do the same?

File: pyproject.toml [hunk 2]
```diff
@@ -29,6 +29,7 @@ dev = [
   "pytest",
   "pytest-cov",
   "requests-mock",
+  "sentry-sdk[flask]",
```
Comment (katzmo) [inline pyproject.toml:32]: Why is sentry a dev dependency, it’s only ever used in production? :thinking:

File: moflask/flask.py (hunk 1 above)
Comment (torotil) [reply moflask/flask.py:52]: image

File: pyproject.toml (hunk 2 above)
Comment (torotil) [reply pyproject.toml:32]: It’s needed to run the tests.

File: moflask/flask.py (hunk 1 above)
Comment (katzmo) [reply moflask/flask.py:52]: Ah, because `config` could override keywords from the `defaults` :bulb:

**Review by katzmo — APPROVED** [review]

File: moflask/flask.py (hunk 1 above)
Comment (torotil) [reply moflask/flask.py:52]: Exactly ;-)

**Pull request merged**